components/link-manager/src/test/kotlin/net/corda/p2p/linkmanager/outbound/OutboundMessageProcessorTest.kt

```diff
@@ -1039,7 +1049,7 @@ class OutboundMessageProcessorTest {
     @Test
     fun `onNext produces a LinkManagerProcessedMarker per message if SessionEstablished`() {
```

Comment:
```suggestion
    fun `onNext produces a LinkManagerProcessedMarker per message if SessionEstablished, but no messages if recordsForSessionEstablished returns no records`() {
```
to make the scenario exercised a bit more explicit?

Reply:
It's too long for detekt (but this is not really the use case, I'm testing that it produced the `LinkManagerProcessedMarker` regardless of the `recordsForSessionEstablished` and in the next test I'm testing the `recordsForSessionEstablished` regardless of the `LinkManagerProcessedMarker`, I'll change the test).